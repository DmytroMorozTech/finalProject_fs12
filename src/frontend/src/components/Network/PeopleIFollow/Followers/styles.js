import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  followers: {
    marginTop: theme.spacing(12)
  },

  peopleFollowers: {
    marginTop: theme.spacing(1),
    border: theme.border.simple,
    borderRadius: theme.shape.medium,
    backgroundColor: theme.palette.common.white,
    display: 'flex'
  }
}))
