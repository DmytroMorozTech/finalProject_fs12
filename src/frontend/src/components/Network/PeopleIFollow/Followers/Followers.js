import React from 'react'
import styles from './styles'
import FollowProfile from '../FollowProfile/FollowProfile'

function Followers () {
  const classes = styles()
  return (
    <div className={classes.peopleFollowers}>
      <FollowProfile/>
    </div>
  )
}

export default Followers
