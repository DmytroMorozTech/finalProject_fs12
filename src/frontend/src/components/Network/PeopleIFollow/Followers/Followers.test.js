import { shallow } from 'enzyme'
import Followers from './Followers'
import React from 'react'

const usersFollowingMe = [
  {
    'id': 4,
    'createdDate': '2015-05-11T22:21:12.123123',
    'firstName': 'Everett',
    'lastName': 'Anderson',
    'fullName': 'Everett Anderson',
    'avatarPublicId': 'linkedin/avatars/i1bzhpqac3i1ld2urju8',
    'profileBgPublicId': '',
    'positionAndCompany': 'Project manager at HQSoftware',
    'email': 'test4@gmail.com',
    'isFollowedByActiveUser': true,
    'numberOfFollowers': 4
  },
  {
    'id': 7,
    'createdDate': '2016-03-04T09:33:31.123123',
    'firstName': 'Connie',
    'lastName': 'Johnston',
    'fullName': 'Connie Johnston',
    'avatarPublicId': 'linkedin/avatars/ob8gmrldzy35dytcib5t',
    'profileBgPublicId': '',
    'positionAndCompany': 'Full-stack developer at Amazon.com',
    'email': 'test7@gmail.com',
    'isFollowedByActiveUser': true,
    'numberOfFollowers': 1
  }
]

jest.mock('./styles', () => () => ({}))

const setUp = (props) => shallow(<Followers {...props}/>)

describe('should render Followers component', () => {
  let component
  beforeEach(() => {
    component = setUp({ usersFollowingMe })
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
