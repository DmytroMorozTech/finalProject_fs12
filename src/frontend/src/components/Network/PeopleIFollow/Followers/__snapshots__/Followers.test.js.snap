// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Followers component to match snapshot 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    variant="h4"
  >
    People who most recently followed you
  </WithStyles(ForwardRef(Typography))>
  <div>
    <FollowProfile
      key="4"
      user={
        Object {
          "avatarPublicId": "linkedin/avatars/i1bzhpqac3i1ld2urju8",
          "createdDate": "2015-05-11T22:21:12.123123",
          "email": "test4@gmail.com",
          "firstName": "Everett",
          "fullName": "Everett Anderson",
          "id": 4,
          "isFollowedByActiveUser": true,
          "lastName": "Anderson",
          "numberOfFollowers": 4,
          "positionAndCompany": "Project manager at HQSoftware",
          "profileBgPublicId": "",
        }
      }
    />
    <FollowProfile
      key="7"
      user={
        Object {
          "avatarPublicId": "linkedin/avatars/ob8gmrldzy35dytcib5t",
          "createdDate": "2016-03-04T09:33:31.123123",
          "email": "test7@gmail.com",
          "firstName": "Connie",
          "fullName": "Connie Johnston",
          "id": 7,
          "isFollowedByActiveUser": true,
          "lastName": "Johnston",
          "numberOfFollowers": 1,
          "positionAndCompany": "Full-stack developer at Amazon.com",
          "profileBgPublicId": "",
        }
      }
    />
  </div>
</div>
`;
